yoti-sdk-sandbox/src/main/java/com/yoti/api/client/sandbox/KeyPairLoader.java

```diff
@@ -0,0 +1,52 @@
+package com.yoti.api.client.sandbox;
+
+import static org.apache.commons.codec.binary.Hex.DEFAULT_CHARSET;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.security.KeyPair;
+
+import com.yoti.api.client.InitialisationException;
+import com.yoti.api.client.KeyPairSource;
+
+import org.bouncycastle.openssl.PEMKeyPair;
+import org.bouncycastle.openssl.PEMParser;
+import org.bouncycastle.openssl.jcajce.JcaPEMKeyConverter;
+
+public class KeyPairLoader {
+
+    public static KeyPair loadKeyPair(KeyPairSource kpSource) throws InitialisationException {
+        try {
+            return kpSource.getFromStream(new KeyStreamVisitor());
+        } catch (IOException e) {
+            throw new InitialisationException("Cannot load key pair", e);
+        }
+    }
+
+    private static class KeyStreamVisitor implements KeyPairSource.StreamVisitor {
```

Comment:
Sure, :+1: it's was done in the first place when I want to make this module completely independent of the others.

Reply:
I have seen that this class is a private class of the SecureYotiClient (not like this class but the same behaviour it's wrapped in a private method). Maybe we can extract them as a common class instead of having two private classes in two modules. What do you think?